### PR TITLE
Add Overkiz documentation for new integration

### PR DESCRIPTION
**Pull request by iMicknl**
## Proposed change
Adds the first version of the Overkiz documentation, for the sensor entity.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62640
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2478/files
- This PR fixes or closes issue: 

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by bdraco — CHANGES_REQUESTED** [review]
The `somfy` changes should probably be in a separate PR after this is merged since this PR is really about `overkiz` and not `somfy`

**Review by bdraco — APPROVED** [review]

**Comment by bdraco** [comment]
@iMicknl For the rest of these, please do one docs PR to match up with the core PR so they are easier to keep track of.  Thanks 👍

File: source/_integrations/overkiz.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Overkiz (by Somfy)
+description: Instructions on how to integrate hubs whom use the Overkiz platform with Home Assistant.
+ha_category:
+  - Hub
+  - Sensor
+  - Button
+  - Lock
+  - Number
+ha_release: 2022.1.0
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:10]: This version is incorrect. It should tag the major version and above all 2022.1 won't be released.

Please open a PR to correct this to 2022.2

File: source/_integrations/overkiz.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+### Z-Wave, Hue and Sonos not supported
+
+Even though most Overkiz hubs supports adding Z-Wave, Hue and Sonos devices, this isn't supported in the Overkiz integration. All these platforms have native integrations in Home Assistant which are more stable and feature rich.
+
+### Overkiz API limits
+
+**Execution queue is full on gateway: #xxxx-xxxx-xxxx (soft limit: 10)**
+
+The Overkiz has a limit of 10 actions that can be executed in parallel. Currently this can only be solved by adding a time-out in between your executions, where you need to take the execution time into account.
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:61]: This sounds like a bug to me, and something that should not be solved in the documentation.

We should not have to burden the end-user with these kinds of things.

File: source/_integrations/overkiz.markdown [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  - '@vlebourl'
+ha_domain: overkiz
+ha_platforms:
+  - sensor
+  - button
+  - lock
+  - number
+---
+
+The Overkiz (by Somfy) integration platform is used by many different vendors, like Somfy, Hitachi and Atlantic. This integration will allow users to integrate their devices into Home Assistant using the Overkiz API.
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:25]: - `like Somfy, Hitachi and Atlantic` -> `like Somfy, Hitachi, and Atlantic`

File: source/_integrations/overkiz.markdown [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+### Overkiz API limits
+
+**Execution queue is full on gateway: #xxxx-xxxx-xxxx (soft limit: 10)**
+
+The Overkiz has a limit of 10 actions that can be executed in parallel. Currently this can only be solved by adding a time-out in between your executions, where you need to take the execution time into account.
+
+**Server busy, please try again later. (Too many executions)**
+
+During peak hours, it could happen that the Overkiz platform is unable to execute your command. The integration will try to retry this command, however this is not guaranteed to succeed.
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:65]: - Space after `succeed. `
- Missing comma after `however`: `this command, however, this is`

File: source/_integrations/overkiz.markdown [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+Over 6000 devices from 60 brands are compatible with the Overkiz platform. This integration will retrieve your devices and map them to the relevant Home Assistant platforms.
+
+{% include integrations/config_flow.md %}
+
+## Known limitations
+
+### Z-Wave, Hue and Sonos not supported
+
+Even though most Overkiz hubs supports adding Z-Wave, Hue and Sonos devices, this isn't supported in the Overkiz integration. All these platforms have native integrations in Home Assistant which are more stable and feature rich.
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:55]: - `supports` -> `support`
- `Z-Wave, Hue and Sonos devices` -> `Z-Wave, Hue, and Sonos devices`
- `feature rich` -> feature-rich`

File: source/_integrations/overkiz.markdown [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+The Overkiz has a limit of 10 actions that can be executed in parallel. Currently this can only be solved by adding a time-out in between your executions, where you need to take the execution time into account.
+
+**Server busy, please try again later. (Too many executions)**
+
+During peak hours, it could happen that the Overkiz platform is unable to execute your command. The integration will try to retry this command, however this is not guaranteed to succeed. 
+
+### Internet connectivity required
+
+This integration communicates via the cloud based Overkiz API, since Overkiz doesn't expose a local API on their hubs. If you are only using Somfy IO compatible devices, you could purchase a Velux KLF200 hub and use to [the Velux integration](/integrations/velux/) which has a local API.
```
Comment (frenck) [inline source/_integrations/overkiz.markdown:69]: - `cloud based` -> `cloud-based`
- Misplaced `to`, remove it: `Velux KLF200 hub and use to` -> `Velux KLF200 hub and use`

**Review by frenck — COMMENTED** [review]
@iMicknl Please address these review comments in a new PR.

Thanks 👍

File: source/_integrations/overkiz.markdown (hunk 2 above)
Comment (bdraco) [reply source/_integrations/overkiz.markdown:61]: @iMicknl AFAICT this is a limitation on Somfy's side with no clear solution. Feel free to reach out on discord to discuss ideas to work around this implementation flaw (on Somfy's side).

**Pull request merged**